src/components/Unlock.vue

```diff
@@ -17,17 +17,17 @@
 		<!-- Unlock input group -->
 		<div id="masterPasswordGroup" v-if="!busy && !isUnlocked()">
 
-			<div class="box-bar small selectable" @click="$router.route('/choose')">
-				<span><b>{{ databaseFileName }}</b> ( click to change <i class="fa fa-database" aria-hidden="true"></i> )</span>
-			</div>
-
 			<div class="unlockLogo stack-item">
 				<img src="assets/icons/logo.png">
 				<span>KeePass Tusk</span>
 			</div>
 
 			<form v-on:submit="clickUnlock">
 
+				<div class="small selectable databaseChoose">
+					<b>{{ databaseFileName }}</b> <span @click="$router.route('/choose')" class="muted-color">change...</span>
```

Comment:
If the entire line is "selectable", it's confusing to have only the change part cause the action.  I moved the router click action up to the parent.